whois/table_rdap_domain.go

```diff
@@ -0,0 +1,63 @@
+package whois
+
+import (
+	"context"
+	"strings"
+
+	"github.com/openrdap/rdap"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableRdapDomain(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "rdap_domain",
+		Description: "RDAP domain information including expiration, DNS servers and contact details.",
+		List: &plugin.ListConfig{
+			KeyColumns: plugin.SingleColumn("domain"),
+			Hydrate:    getRdapDomain,
+		},
+		Columns: []*plugin.Column{
+			// Top columns
+			{Name: "domain", Type: proto.ColumnType_STRING, Transform: transform.FromQual("domain"), Description: "Domain name the RDAP information relates to."},
+
+			{Name: "domain_id", Type: proto.ColumnType_STRING, Transform: transform.FromField("Handle").NullIfZero(), Description: "Unique identifier for the domain."},
```

Comment:
Should the column be called `handle`, if the field name is `Handle`? Is there a reason you chose `domain_id` instead? Also, where does the description come from?

Reply:
@cbruno10, I've renamed the column domain_id to handle to ensure consistency with the API response. For most column descriptions, I referred to [this documentation](https://fred.nic.cz/documentation/html/RDAPReference/Domain.html). For those not covered in the documentation, I searched online and selected descriptions that best matched the data contained in each column.

https://datatracker.ietf.org/doc/html/rfc7483.html#section-5.1
https://datatracker.ietf.org/doc/html/rfc7483#section-5.3